packages/app/src/app/overmind/internalActions.ts

```diff
@@ -590,61 +591,114 @@ export const setViewModeForDashboard = ({ effects, state }: Context) => {
   }
 };
 
-export const setActiveTeamFromUrlOrStore = async ({ actions }: Context) =>
-  actions.internal.setActiveTeamFromUrl() ||
-  actions.internal.setActiveTeamFromLocalStorage() ||
-  actions.internal.setActiveTeamFromPersonalWorkspaceId();
+const INVALID_ID_TITLE = 'Workspace not recognized.';
+const INVALID_ID_MESSAGE =
+  "The workspace in the URL or stored in your browser is unknown. We've automatically switched to your personal workspace.";
 
-export const setActiveTeamFromLocalStorage = ({
+// Rename to initializeTeam (because we also use personalWorkspaceId);
+export const setActiveTeamFromUrlOrStore = async ({
```

Comment:
```suggestion
export const setActiveWorkspaceFromUrlOrStore = async ({
```

Works for personal and team.